### DISCUSSION TITLE
Register shortcuts by character not key

### DISCUSSION THREAD

**Discussion by dblbirdie** [rxhanson/Rectangle#919] [Ideas]
I use Rectangle on my laptop. Sometimes with external keyboard, sometimes not. If I record a shortcut using `-` (minus) on the numpad, it is not accessible on a keyboard without numpad because `-` (dash) is a different key (even though pressing it results in the same character).

I see the following options:
1. Additional shortcut via macos preferences (did not work)
2. Register shortcuts by character not by key
3. Enable registering more than one shortcut for a action

What is your opinion? Thanks in advance.

**Comment by rxhanson** [comment]
Thanks for the feedback. Fixing issues like this are likely to add some unintuitive complexity to the UI of the app, which means that a fix would likely be a hidden command. I don't have plans to add something like this in myself, but I don't see a reason to turn down a good pull request with a good way to do this. 

If you're looking for an alternative (paid) way to achieve this that already exists, you could try creating a custom shortcut in [Rectangle Pro](https://rectangleapp.com/pro).
